exercises/practice/gigasecond/test/gigasecond_test.exs

```diff
@@ -2,20 +2,30 @@ defmodule GigasecondTest do
   use ExUnit.Case
 
   # @tag :pending
-  test "from 4/25/2011" do
+  test "from 2011-04-25 00:00:00" do
```

Comment:
More international and programmer-friendly date format...

Reply:
nice usability change